src/compiler/scala/tools/nsc/symtab/SymbolLoaders.scala

```diff
@@ -15,6 +15,8 @@ package symtab
 
 import classfile.ClassfileParser
 import java.io.IOException
+
+import scala.language.reflectiveCalls
```

Comment:
it's because we call `symbolTable.settings`, which is a member of the refinement type `SymbolTable { def settings: scala.tools.nsc.Settings }` whereas `SymbolTable` itself has `def settings: MutableSettings`, a supertype of `nsc.Settings`.

Thankfully scalac is smarter than intellij and realizes that it can invoke `settings` using `SymbolTable`'s method and then `checkcast` the result to (for instance) `ScalaSettings`, the owner of `debug`:

```scala
  protected def signalError(root: Symbol, ex: Throwable) {
    if (settings.debug) ex.printStackTrace()
    /* ... */
```

```
         0: getstatic     #193                // Field scala/reflect/internal/settings/MutableSettings$.MODULE$:Lscala/reflect/internal/settings/MutableSettings$;
         3: aload_0
         4: invokevirtual #140                // Method symbolTable:()Lscala/tools/nsc/symtab/SymbolTable;
         7: invokevirtual #197                // Method scala/tools/nsc/symtab/SymbolTable.settings:()Lscala/reflect/internal/settings/MutableSettings;
        10: checkcast     #199                // class scala/tools/nsc/settings/ScalaSettings
        13: invokeinterface #203,  1          // InterfaceMethod scala/tools/nsc/settings/ScalaSettings.debug:()Lscala/tools/nsc/settings/MutableSettings;
```

so I think we don't have to worry here.